.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,16 +2,17 @@
 Title
 ---
 
-**Important note**
+**Important notes**
 Please note that only PrestoN is able to merge Pull Requests into master.
+Please note that if u incorrectcly filled out or not filled out the PR template, it will result in the closure of this PR.
 
 **Pull Request Type**
 Please select what type of pull request this is:
 - [ ] Bugfix
 - [ ] Feature Implementation
 
 **Related issue**
-Please link the issue your pull request is referring to.
+Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #99999". If this pull request fully resolves multiple relevant issues, put "closes" before both issue numbers. Example: "closes #99999, closes #12345".
```

Comment:
```suggestion
Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #123456".
```

I think your second added sentence is not necessary given that this has only rarely been a problem.

Reply:
Yeah that makes sense. I guess i just wanted to be clear about the way it was done.